src/components/grouped-character/grouped-character.pw.tsx

```diff
@@ -0,0 +1,496 @@
+        specificValue
+      );
+    });
+  });
+
+  specialCharacters.forEach((specificValue) => {
+    test(`should check fieldHelp renders properly with ${specificValue} specific value`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<GroupedCharacterComponent fieldHelp={specificValue} />);
+
+      await expect(getDataElementByValue(page, "help")).toHaveText(
+        specificValue
+      );
+    });
+  });
+
+  specialCharacters.forEach((specificValue) => {
+    test(`should check tooltip renders properly with ${specificValue}s specific values`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<GroupedCharacterComponent labelHelp={specificValue} />);
+
+      const question = getDataElementByValue(page, "question");
+      await question.hover();
+      await expect(tooltipPreview(page)).toHaveText(specificValue);
+    });
+  });
+
+  test(`should check add icon inside of the GroupedCharacter component renders`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<GroupedCharacterComponent inputIcon="add" />);
+
+    await expect(getDataElementByValue(page, "add")).toBeVisible();
+  });
+
+  test(`should check the GroupedCharacter component is disabled`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<GroupedCharacterComponent disabled />);
+
+    const input = getDataElementByValue(page, "input").locator("..");
+    await expect(input).toHaveAttribute("disabled", /.*/);
+    await expect(getDataElementByValue(page, "input")).toBeDisabled();
+  });
+
+  test(`should check the GroupedCharacter component is required`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<GroupedCharacterComponent required />);
+
+    await verifyRequiredAsteriskForLabel(page);
+  });
+
+  test(`should check the GroupedCharacter component has autofocus`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<GroupedCharacterComponent autoFocus />);
+
+    await expect(getDataElementByValue(page, "input")).toBeFocused();
+  });
+
+  ([
+    ["right", "end"],
+    ["left", "start"],
+  ] as [GroupedCharacterProps["labelAlign"], string][]).forEach(
+    ([alignment, cssProp]) => {
+      test(`should use ${alignment} as labelAlignment and render it with ${cssProp} as css properties`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <GroupedCharacterComponent labelInline labelAlign={alignment} />
+        );
+
+        const labelParent = getDataElementByValue(page, "label").locator("..");
+        await expect(labelParent).toHaveCSS("-webkit-box-pack", cssProp);
+        await expect(labelParent).toHaveCSS(
+          "justify-content",
+          `flex-${cssProp}`
+        );
+      });
+    }
+  );
+});
+
+test.describe("Props, event & styling tests", () => {
+  ["10%", "30%", "50%", "80%", "100%"].forEach((maxWidth) => {
+    test(`should check maxWidth as ${maxWidth} for GroupedCharacter component`, async ({
```

Comment:
**non-blocking**: it might be better to change the description to `should render properly with maxWidth of ${maxWidth}
`